help/deploy/complete-setup/configure-auth.md

```diff
@@ -3,75 +3,74 @@ title: "Configure authentication"
 sidebar_position: 1
 ---
 
-Out-of-the-box, Datagrok offers many authentication methods, including popular methods such as SSO and OAuth.
+Datagrok supports authentication using:
 
-To accommodate any enterprise needs, we can develop a customer-specific integration.
+* [Login-password](#login-password-authentication)
+* [Oauth](#oauth-authentication)
+* [OpenID](#openid-authentication)
+* [LDAP](#ldap-authentication)
 
-All methods can be enabled separately or combined.
+You can enable all authentication methods separately or combined. To accommodate any enterprise needs, we can develop a customer-specific integration. 
 
 [Learn more about authentification](../govern/authentication.md).
 
 ## Common options
 
-If you want to display something on the login window, set the 'Prompt' option
-![Login prompt](login-prompt.png)
+To display something on the login window, set the **Prompt** option
+<p align="center">
+  <img src="login-prompt.png" alt="Login prompt" width="300"></img>
+</p>
```

Comment:
Docusaurus can not load the image
`<img alt="login-prompt" src={require('./login-prompt.png').default} width="300px"/>`
This way it will work